OMEdit/OMEdit/OMEditGUI/Options/OptionsDialog.cpp

```diff
@@ -1680,6 +1684,20 @@ void OptionsDialog::reject()
   QDialog::reject();
 }
 
+void OptionsDialog::reset()
+{
+  const QString title = "Reset to default";
+  const QString text = "Are you sure that you want to reset OMEdit? This operation can not be undone. Please back up your settings file before proceeding, restart to have the changes take effect.";
```

Comment:
Maybe the normal users don't know where the settings file is located. Can we add the file path here?
https://doc.qt.io/qt-5/qsettings.html#fileName

Reply:
Good point, I guess having "file" above as a hyperlink in the popup which opens the folder if the user clicks it, thoughts?